### DISCUSSION TITLE
Cannot post new issue for `Homebrew/brew` project (HTTP 400 response)

### DISCUSSION THREAD

**Discussion by AmokHuginnsson** [community/community#7788] [Projects and Issues]
I try to post new issue for [brew](https://github.com/Homebrew/brew/issues/new?assignees=&labels=bug&template=bug.yml) project.
I have read CONTRIBUTING.md and I carefully followed all the steps and filled all template inputs and checked all the checkboxes.
When I try to post filled out form for a bug (I click "Submit new issue" button) a blank page appears and browser inspector shows "400 Bad Request".

I am at a loss what I could be doing wrong. I tried both Firefox and Google Chrome on Ubuntu 21.10.

Please advise.

Thank you.

**Comment by luckman212** [comment]
Having the same issue. Not sure how to shine any light on it.

**Reply by luckman212** [reply]
GH ticket #1409253

**Comment by luckman212** [comment]
Apparently this is fixable by https://github.com/Homebrew/discussions/discussions/2526#discussioncomment-1731168
